frontend-react/src/components/MessageTracker/MessageDetails.tsx

```diff
@@ -15,57 +19,132 @@ type MessageDetailsProps = {
     id: string | undefined;
 };
 
+const dataToAccordionItems = (props: {
```

Comment:
This function feels a bit odd to me, but I can't think of a better way to pass the necessary data and the custom WarningsErrors component to the Accordion right now 🤔